app/models/miq_request.rb

```diff
@@ -35,12 +35,16 @@ class MiqRequest < ApplicationRecord
   virtual_column  :stamped_on,           :type => :datetime, :uses => :miq_approvals
   virtual_column  :v_allowed_tags,       :type => :string,   :uses => :workflow
   virtual_column  :v_workflow_class,     :type => :string,   :uses => :workflow
+  virtual_column  :v_supports_pxe?,      :type => :string,   :uses => :workflow
+  virtual_column  :v_supports_iso?,      :type => :string,   :uses => :workflow
   virtual_column  :request_type_display, :type => :string
   virtual_column  :resource_type,        :type => :string
   virtual_column  :state,                :type => :string
 
   delegate :allowed_tags,                :to => :workflow,   :prefix => :v,  :allow_nil => true
   delegate :class,                       :to => :workflow,   :prefix => :v_workflow
+  delegate :supports_pxe?,               :to => :workflow,   :prefix => :v
```

Comment:
@kbrock For `virtual_delegate` to work correctly in this case, do we need a `belongs_to` association in `MiqRequestWorkflow` : 
```ruby
belongs_to  :miq_request
```

(Although note that  `MiqRequestWorkflow` is not ActiveRecord)

I've tried various things so far, and I'm seeing errors of this nature --
```
ArgumentError (Delegation needs an association. Supply an options hash with a :to key as the last argument (e.g. delegate :hello, to: :greeter).):
```
```
ActionController::RoutingError (uninitialized constant MiqRequest::Workflow)
```
```
ActionController::RoutingError (undefined method `type_for_attribute' for MiqRequestWorkflow:Class)
```